conf/modules.config

```diff
@@ -1692,16 +1713,17 @@ process {
     }
 
     withName: GATK_UNIFIEDGENOTYPER {
-        tag = { "${meta.reference}|${meta.sample_id}" }
+        tag = { params.genotyping_use_unmerged_libraries ? "${meta.reference}|${meta.sample_id}|${meta.library_id}" : "${meta.reference}|${meta.sample_id}" }
         ext.args = {[
-            "--sample_ploidy ${meta2.ploidy}",
+            // ploidy set at 2 for non-specified ploidy in input fasta sheet for consistency with gatk default
+            meta2.ploidy ? "--sample_ploidy ${meta2.ploidy}" : "--sample_ploidy 2",
```

Comment:
I think we should use the fasta schema to provide a default value for this instead of hard-coding default values for ploidy in each module.